setup/instance/upgrade.go

```diff
@@ -76,7 +76,7 @@ func assertPatchUpgrade(currentVersion, desiredVersion string) error {
 
 	current, err := semver.Make(currentVersion)
 	if err != nil {
-		return fmt.Errorf("Failed to parse current version: %v", err)
+		return fmt.Errorf("Current version not semver compatible: %v\nUse --force to override this behavior", currentVersion)
```

Comment:
Let's try to keep the line length a bit smaller and use the error message, e.g.:
```
text := fmt.Sprintf("Failed to parse current version ('%s'): %v\n", currentVersion, err)
text += "Use --force to disable semantic version checking"
return fmt.Errorf(text)
```